Lib/ufo2ft/fastUFO.py

```diff
@@ -192,32 +247,71 @@ def __init__(
             self.groups = reader.readGroups()
             self.kerning = reader.readKerning()
             self.lib = reader.readLib()
-            glyphset = reader.getGlyphSet()
-            for name in glyphset.keys():
-                glyph = Glyph(name, self)
-                glyphset.readGlyph(
-                    glyphName=name, glyphObject=glyph, pointPen=glyph)
-                self.glyphset[name] = glyph
+            defaultLayerName = reader.getDefaultLayerName()
+            for layerName in reader.getLayerNames():
+                glyphset = reader.getGlyphSet(layerName)
+                if layerName not in self.layers:
+                    self.newLayer(layerName)
+                for name in glyphset.keys():
+                    glyph = self.layers[layerName].newGlyph(name)
+                    glyphset.readGlyph(
+                        glyphName=name, glyphObject=glyph, pointPen=glyph)
```

Comment:
is it really necessary to read all the glyphs from all the layers?
This was supposed to be a fast, drop-in replacement for defcon to be used by fontmake for the sake of building binary fonts. The default layer is the only one that gets built. Or am I missing something?